pal/screen-adapter/minigame/screen-adapter.ts

```diff
@@ -156,11 +156,9 @@ class ScreenAdapter extends EventTarget {
     constructor () {
         super();
         // TODO: onResize or onOrientationChange is not supported well
-        if (WECHAT || WECHAT_MINI_PROGRAM || RUNTIME_BASED) {
-            minigame.onWindowResize?.(() => {
-                this.emit('window-resize', this.windowSize.width, this.windowSize.height);
-            });
-        }
+        minigame.onWindowResize?.(() => {
```

Comment:
Does all mini game platforms support resize event well?

Reply:
If any platform does not support it well, it should be a hidden problem that needs to be fixed

Reply:
Any issue for it?